modeldb/commands.py

```diff
@@ -32,16 +47,28 @@ def runmodels(args=None):
     Options:
         --gout                  Include gout into the report. Note that gout data can be very big, so disabled by default.
         --virtual               Run in headless mode. You need a back-end like Xvfb.
+        --clean                 Auto-clean output directory before running (useful for consecutive runs)
 
     Examples
         runmodels /path/to/workdir
-        runmodels /path/to/workdir 23613 12344
+        runmodels --clean /path/to/workdir 23613 12344
     """
     options = docopt(runmodels.__doc__, args)
     working_dir = options.pop("<WorkingDirectory>")
     model_ids = [int(model_id) for model_id in options.pop("<model_id>")]
     gout = options.pop("--gout", False)
     virtual = options.pop("--virtual", False)
+    clean = options.pop("--clean", False)
+
+    if not clean and is_dir_non_empty(working_dir):
+        print("WARNING: WorkingDirectory {} exists and is non empty.".format(working_dir))
+        print("\tre-run with --clean if you wish to overwrite!")
+        sys.exit(1)
+
+    if clean:
+        shutil.rmtree(working_dir)
```

Comment:
apps doing `rmtree or rm -rf` are bit scary. Any chance that this could ruins user directory accidentally? (e.g. if user is in the wrong dir?)

Reply:
yeah, like always. maybe it's better if we turn the working dir into an option, i.e. `--workdir`

Reply:
or add a marker file, `.nrn-modeldb-ci`

Reply:
Ok so I've added a `--workdir` and `--clean` will simply `rmtree` the model folder.